tests/unit/notification/NotificationConfigManager.js

```diff
@@ -118,6 +124,17 @@ describe('NotificationConfigManager', () => {
                 next => checkParentConfigZkNode(manager, next),
             ], done);
         });
+
+        it('should return error if checkConfigParentNode fails', done => {
+            sinon.stub(NotificationConfigManager.prototype,
+                '_checkConfigurationParentNode')
+                .callsFake(checkConfigParentNodeStub);
+            const manager = new NotificationConfigManager(params);
+            manager.init(err => {
+                assert(err);
```

Comment:
```suggestion
                assert.ifError(err);
```
https://nodejs.org/api/assert.html#assert_assert_iferror_value

Reply:
The test is expecting an error to be returned.